### DISCUSSION TITLE
how to handle nuxt-edge

### DISCUSSION THREAD

**Discussion by gabrielsze** [nuxt/nuxt#9513] [Questions (Nuxt 2)] [answered]
i'm trying to replace nuxt with nuxt-edge, but ran into a npm issue

`No matching version found for nuxt-edge@^2.14.0.`

 Is there a page to see the latest ones, or whats the best practice in handling nuxt-edge versioning?

**Comment by TheAlexLichter** [comment, marked as answer]
Nuxt Edge Versioning is a little different. The format is MAJOR.MINOR.PATCH-DATE.COMMITHASH (see https://github.com/nuxt/nuxt.js/blob/aa5d05f8b9815de15673cb3174a590cfb65cefd6/scripts/package.js#L105 as reference) hence the expression doesn't match.

See https://www.npmjs.com/package/nuxt-edge?activeTab=versions for the full list of available versions.
